src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -5,16 +5,22 @@ import session from 'services/session';
 import styles from '../Doughnut.module.css';
 import { Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
 import { theme360 } from 'theme';
+import { format, parseISO } from 'date-fns';
 
 const DaysLeft = () => {
   const [daysRemaining, setDaysRemaining] = useState();
   const [daysFinished, setDaysFinished] = useState();
   const [currentSessionDescription, setCurrentSessionDescription] = useState();
   const [loading, setLoading] = useState(true);
+  const [firstDay, setFirstDay] = useState('');
+  const [lastDay, setLastDay] = useState('');
 
   useEffect(() => {
     const load = async () => {
       setLoading(true);
+      const currSession = await session.getCurrent();
+      const firstDay = format(parseISO(currSession.SessionBeginDate), 'MM/dd/yyyy');
+      const lastDay = format(parseISO(currSession.SessionEndDate), 'MM/dd/yyyy');
```

Comment:
```suggestion
      const first = format(parseISO(currSession.SessionBeginDate), 'MM/dd/yyyy');
      const last = format(parseISO(currSession.SessionEndDate), 'MM/dd/yyyy');
```

to prevent confusion, you should use a different local variable name than what you used for your state variable

Reply:
also might be helpful to just make an auxiliary function to format since it's done identically twice:
```
const formatter = (iso) => return format(parseISO(iso)), 'MM/dd/yyyy')
```